src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -438,7 +440,10 @@ class Message(pydantic.BaseModel):
     Single line content.
     """
 
-    ray_filename: RayFilename
+    # Pydantic v2 lets us do multiple aliases
+    # TODO: ORQSDK-956 - update Pydantic v2 and use AliasChoice
+    log_filename: Optional[LogFilename] = None
```

Comment:
**issue**: there's no `log_filename` in the schema for [TaskRunLogs](https://github.com/zapatacomputing/workflow-driver/blob/17a53d42f42f1f90ee1c1c1eba2032117a5380e5/openapi/src/resources/task-run-logs.yaml#L28) nor [WorkflowRunLogs](https://github.com/zapatacomputing/workflow-driver/blob/17a53d42f42f1f90ee1c1c1eba2032117a5380e5/openapi/src/resources/workflow-run-logs.yaml#L26) entries.

Reply:
TaskRunLogs are out of date, Emre added this last night.